packages/block-library/src/post-title/edit.js

```diff
@@ -1,10 +1,32 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { useSelect } from '@wordpress/data';
+import {
+	AlignmentToolbar,
+	BlockControls,
+	__experimentalBlock as Block,
```

Comment:
Using this block wrapper means we need to add "lightBlockWrapper" support flag as well (it's opt-in)